README.md

```diff
@@ -1,8 +1,14 @@
-# Okta + Android (Java) & Okta-Hosted Login Page Example
+# Okta + Android (Kotlin) & Okta-Hosted Login Page Example
 
-This sample shows how to use the Okta Android SDK to sign a user in to an Android application built in Java. The process uses Authorization Code Flow + PKCE, where the user is redirected to Okta for authentication. After the user authenticates, they are redirected back to the application with an authorization code, which is exchanged for an ID token and access token.
+This sample shows how to use the Okta Android SDK to sign a user in to an Android application built in Kotlin. The process uses Authorization Code Flow + PKCE, where the user is redirected to Okta for authentication. After the user authenticates, they are redirected back to the application with an authorization code, which is exchanged for an ID token and access token.
```

Comment:
Some suggested rewording for Okta style, clarity, and flow (notes the changes follow). The second paragraph feels like it needs more work.

```suggestion
Authenticate a user using the Redirect model in your Kotlin Android app using the Okta Android SDK.

The app launches a sign-in view hosted by Okta that handles the authorization flow. Control is returned to the app once the user finishes the flow. An authorization code is returned to the app after a successful sign on. This code is then exchanged for ID and access tokens.
```

"This sample" isn't needed as it's clear from context. This lets the developer find out what the sample is about sooner and figure out if they needed it.
"Redirect model" is the way the other docs talk about this type of authentication
I structured it as two paragraphs. The first is a one sentence abstract summarizing what the user can do with the sample/what question it answers. The second gives an overview of the process